fmts/haskell.go

```diff
@@ -0,0 +1,19 @@
+package fmts
+
+func init() {
+	const lang = "haskell"
+
+	register(&Fmt{
+		Name: "hlint",
+		Errorformat: []string{
+			`%f:%l:%c: %m`,
+			`%A%f:%l:%c: %.%#: %m`,
+			`%Z%p^%#`,
+			`%C%.%#`,
+			`%-G%.%#`,
+		},
+		Description: "Linter for Haskell soudrce code",
```

Comment:
```suggestion
		Description: "Linter for Haskell source code",
```

Reply:
whoops. My bad. I've fixed this now @mgrachev .